README.md

```diff
@@ -1,32 +1,505 @@
-What is Swrve
--------------
-Swrve is a single integrated platform delivering everything you need to drive mobile engagement and create valuable consumer relationships on mobile.  
-This native iOS SDK will enable your app to use all of these features.
+Swrve
+=
+Swrve is a single integrated platform delivering everything you need to drive mobile engagement and create valuable consumer relationships on mobile.
 
-Getting started
----------------
-Have a look at the quick integration guide at http://docs.swrve.com/developer-documentation/37926233/
+This iOS SDK will enable your app to use all of these features.
 
-Installing using CocoaPods
---------------------------
-Add the following line to your Podfile
-```
-pod ‘SwrveSDK’
-```
+- [Getting started](#getting-started)
+  - [Requirements](#requirements)
+    - [Xcode (latest)](#xcode-latest)
+    - [Frameworks](#frameworks)
+    - [Automatic reference counting](#automatic-reference-counting)
```

Comment:
Perhaps a note in the Getting Started section about using a Sandbox until finished QA, then moving over to Production? Had a few instances where a. people didn't realise/forgot to move over, b. people ask to delete test data from a production dashboard. A paragraph explaining this flow at the beginning could solve both these issues.